integration/common/openlineage/common/provider/dbt.py

```diff
@@ -417,14 +438,16 @@ def node_to_output_dataset(self, node: ModelNode, has_facets: bool = False) -> O
                 node.catalog_node,
                 [
                     ['stats', 'num_bytes', 'value'],  # bigquery
-                    ['stats', 'bytes', 'value']  # snowflake
+                    ['stats', 'bytes', 'value'],  # snowflake
+                    ['stats', 'size', 'value']  # redshift (Note: size = count of 1MB blocks)
```

Comment:
We need to multiply this then - it needs to be consistent with rest of databases